content/applications/websites/livechat/overview/get_started.rst

```diff
@@ -2,82 +2,200 @@
    :align: center
-   :alt: View of a channel’s rules form for Odoo Live Chat
+   :alt: View of a channel's rules form for Odoo Live Chat
+
+Add the URL for the pages this channel will be applied to in the :guilabel:`URL Regex` field. If
+this channel will only be available to users in specific countries, add them to the
+:guilabel:`Country` field. If this field is left blank, the channel will be available to all site
+visitors.
 
 .. note::
-   GeoIP, which refers to the process of finding a computer terminal’s geographical location by its
-   IP address, must be installed on your server. Otherwise, under *Channel Rules*, countries are
-   not taken into account.
+   In order to track the geographical location of visitors, :guilabel:`GeoIP` must be installed on
+   the database. While this feature is installed by default on :guilabel:`Odoo Online`,
+   :guilabel:`On-Premise` databases will require additional :doc:`setup steps
+   </applications/websites/website/publish/on-premise_geo-ip-installation>`.
+
+Widget
+------
+
+The :guilabel:`Widget` tab on the live chat channel form offers an embeddable website widget, or a
+shortcode for instant customer/supplier access to a live chat window.
 
-External options
-----------------
+The live chat :guilabel:`Widget` can be added to websites created through Odoo by navigating to
+the :menuselection:`Website --> Configuration --> Settings`. Then scroll to the :guilabel:`Live
+Chat` section, and select the channel to add to the site. Click :guilabel:`Save` to apply.
 
-| If your website was not created with Odoo, you can find the code to be added to your own, under
-  the *Widget* tab.
-| Odoo also offers an URL you can send to users so they can have access to a single live chat page.
+To add the widget to a website created on a third-party platform, click :guilabel:`COPY` and paste
+the code into the `<head>` tag on the site.
 
-.. image:: get_started/widget.png
+Likewise, to send a live chat session to a customer or supplier, click the second :guilabel:`COPY`
+button which contains a link to join directly.
+
+.. image:: get_started/live-chat-widget-code.png
    :align: center
-   :alt: View of the widget tab for Odoo Live Chat
+   :alt: View of the widget tab for Odoo Live Chat.
+
+Participate in live chat
+========================
+
+As explained above, *operators* are the users who will respond to live chat requests from
+customers. The information below outlines the necessary steps for operators participating in
+live chat conversations on an *Odoo* database.
 
-Managing chat requests
-======================
+Set an online chat name
+-----------------------
 
-Conversations initiated by visitors pop up as a direct message, and are shown in *Discuss*.
-Therefore, inquiries can be answered wherever you are in Odoo.
+Before participating in a live chat, operators should update their :guilabel:`Online Chat Name`.
+This is the name that will be displayed to site visitors in the live chat conversation.
```

Comment:
Again, I realize it can seem arbitrary, however I try to keep it separate between actual UI items (which require the guilabel) and concepts (which do not). I will make sure to take another look to ensure that there is consistency between these two, however, I have been told not to add a guilabel unless specifically referring to a UI item.